test/table_unspanner_test.rb

```diff
@@ -1,11 +1,32 @@
 require 'test_helper'
 
-class TableUnspannerTest < Minitest::Test
-  def test_that_it_has_a_version_number
-    refute_nil ::TableUnspanner::VERSION
+describe 'TableUnspanner' do
+  it 'has a version number' do
+    ::TableUnspanner::VERSION.wont_be_nil
   end
 
-  def test_it_does_something_useful
-    assert false
+  describe 'UnspannedTable' do
+    it 'replaces span elements with multiple cells' do
+      table = <<-TABLE
+      <table>
+        <tr>
+          <th>Name</th>
+          <th>Role</th>
+        </tr>
+        <tr>
+          <td>Alice</td>
+          <td rowspan="2">Test subject</td>
+        </tr>
+        <tr>
+          <td>Bob</td>
+        </tr>
+      </table>
+      TABLE
+
```

Comment:
I think you need a test that has at least one colspan, and probably at least one span in a `th` as well.